server/views/global/palette.config.yml

```diff
@@ -33,6 +33,8 @@ context:
     hex: "#717171"
   - name: accessible-grey-2
     hex: "#8f8f8f"
+  - name: smoke
+    hex: "#e8e8e8"
```

Comment:
As somebody who‘s just getting up-to-speed on the codebase, it seems weird that these colour definitions are duplicated.

Unless one is auto-generated from the other? In which case:

* Boo hiss checking in autogen output.
* There should be a comment at the top of the autogen saying “This file is autogenerated, look here to make edits”.